committeeoversightapp/models.py

```diff
@@ -94,10 +88,215 @@ def chair(self):
     def hide_rating(self):
         return CommitteeDetailPage.objects.get(committee=self.id).hide_rating
 
+    @property
+    def ratings_by_congress_desc(self):
+        return self.committeerating_set.all().order_by('-congress__id')
+
+    @property
+    def ratings_by_congress_asc(self):
+        return self.committeerating_set.all().order_by('congress__id')
+
+    @property
+    def latest_rating(self):
+        return self.ratings_by_congress_desc[0]
+
+    @property
+    def max_chp_points(self):
+        return self.ratings_by_congress_desc.aggregate(
+            Max('chp_points')
+        )['chp_points__max']
+
+    @property
+    def investigative_oversight_hearings_max(self):
+        return self.get_max_rating('investigative_oversight_hearings')
+
+    @property
+    def policy_legislative_hearings_max(self):
+        return self.get_max_rating('policy_legislative_hearings')
+
+    @property
+    def total_hearings_max(self):
+        return self.get_max_rating('total_hearings')
+
+    @property
+    def investigative_oversight_hearings_avg(self):
+        return self.get_avg_rating('investigative_oversight_hearings')
+
+    @property
+    def policy_legislative_hearings_avg(self):
+        return self.get_avg_rating('policy_legislative_hearings')
+
+    @property
+    def total_hearings_avg(self):
+        return self.get_avg_rating('total_hearings')
+
+    def get_avg_rating(self, hearing_type):
+        return round(self.ratings_by_congress_desc \
+            .aggregate(
+                Avg(hearing_type)
+            )[hearing_type + '__avg'], 2)
+
+    def get_max_rating(self, hearing_type):
+        return round(self.ratings_by_congress_desc \
+            .aggregate(
+                Max(hearing_type)
+            )[hearing_type + '__max'], 2)
+
     def __str__(self):
         return self.name
 
 
+class Congress(models.Model):
+    start_date = models.DateField()
+    end_date = models.DateField()
+
+    @property
+    def label(self):
+        '''
+        116 => '116th Congress'
+        '''
+        return '{} Congress'.format(ordinal(self.id))
+
+    @property
+    def is_current(self):
+        if self.start_date < date.today() < self.end_date:
+            return True
+        else:
+            return False
+
+    @property
+    def percent_passed(self):
+        days_in_session = 668
+        days_passed = (date.today() - self.start_date).days
+        percent_passed = round(days_passed * 100.0 / days_in_session, 2)
```

Comment:
Given https://github.com/datamade/committee-oversight/pull/117#discussion_r349276996, what do you think about reordering?